.github/workflows/build.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         if: steps.cache-pact.outputs.cache-hit != 'true'
         run: |
           cd ~
-          curl -fsSL https://raw.githubusercontent.com/pact-foundation/pact-ruby-standalone/master/install.sh | bash
+          curl -fsSL https://raw.githubusercontent.com/pact-foundation/pact-ruby-standalone/master/install.sh | PACT_CLI_VERSION=v2.4.2 sh
```

Comment:
I'll try to move the `PACT_CLI_VERSION` to an env var in the next iteration.

https://docs.github.com/en/actions/learn-github-actions/variables